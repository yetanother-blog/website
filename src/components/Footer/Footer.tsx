import React from 'react';
import { Box } from '../Box/Box';

export const Footer: React.FC = ({ children }) => {
  return (
    <Box
      as="footer"
      display="flex"
      flexDirection={['column', 'row']}
      justifyContent="flex-end"
      maxWidth="940px"
      width="100%"
      minHeight="60px"
    >
      {children}
    </Box>
  );
};
