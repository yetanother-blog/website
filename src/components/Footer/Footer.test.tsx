import React from 'react';
import { render } from '@testing-library/react';
import { Footer } from './Footer';

describe('Footer', () => {
  it('should render w/ children', () => {
    const { container } = render(<Footer>some footer content</Footer>);
    expect(container).toMatchSnapshot();
  });
});
