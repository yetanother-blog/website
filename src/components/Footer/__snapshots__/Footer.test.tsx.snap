// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render w/ children 1`] = `
<div>
  <footer
    class="sc-bdnxRM cCBjOh"
    display="flex"
    width="100%"
  >
    some footer content
  </footer>
</div>
`;
